src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -62,6 +62,11 @@ internal abstract class PENamedTypeSymbol : NamedTypeSymbol
         /// </summary>
         private TypeKind _lazyKind;
 
+        /// <summary>
+        /// Lazily initiated HasEmbeddedAttribute
+        /// </summary>
+        private ThreeState _lazyHasEmbeddedAttribute = ThreeState.Unknown;
```

Comment:
Noted. Will fix it now.

---
In reply to: [114065603](https://github.com/dotnet/roslyn/pull/18715#discussion_r114065603) [](ancestors = 114065603)